### ISSUE TITLE
Question: Is a update every 10 second to be expected?

### ISSUE THREAD

**Issue by mariwing** [gcobb321/icloud3#100]
Hi and thank you for all the work on this component!

I am seeing in the log that this "pattern" repeats every 10 seconds for many hours:

15:27:00 | Outside Zone and No Exit Zone trigger, Keeping in zone > Zone-home, Distance-42.84m, DiscardDist-25m to 125m
15:27:00 | iOSApp Trigger Change detected > iOSApp Loc Update
15:26:50 | Outside Zone and No Exit Zone trigger, Keeping in zone > Zone-home, Distance-42.84m, DiscardDist-25m to 125m
15:26:50 | iOSApp Trigger Change detected > iOSApp Loc Update

My phone is laying still on my home office desk all day long and is in "real life" in the home zone all the time

It stops after I launch the app and refresh it. This means that the app was not open and maybe not even connected to my Home Assistant instance. Do you have any advice or idea what is going on here and what (if anything) I could do to stop this?

Thank you again :)

**Comment by gcobb321** [comment]
The iOS app issues a zone exit trigger when the distance from the zone is 100m, regardless of the actual size where iCloud3 looks at the actual size. This can lead to zone size mismatches between the iOS app and iCloud3. Sometimes the iOS app sends an exit trigger, sometimes it didn’t.

What is the size of your home zone? Try making it 100m and see if the problem goes away.

————————
Gary Cobb

**Comment by mariwing** [comment]
Thank you for the quick reply and good explanation.

I have set my home zone to 25 meters because I live quite near my mother and when I am visiting her I didn't want my status to be Home. I will try to set the zone to 100 meters and see if I can live with calling my mothers house my home ;-)

**Comment by gcobb321** [comment]
You may be able to set a smaller zone on your mother's house. Then when you are there, iCloud3 will put you in that zone. The iOS App may not cooperate, you'll just have to see if it does.

**Comment by mariwing** [comment]
Thanks for more suggestions!

An aditional challenge is that my mother also have the home assistant app (to keep an eye on me and her grandchildren...) so when my home zone overlapped here house she ended up as being registrered in my house which in turns broke automations like start the vacuum when nobody is home.

I now have made a zone for her house at 25 meter radius and have my home radius at 100 meters. I have moved my home zone "of center" so my house is not in the center of the circle and therefore the two zones doesn't overlap longer. 

It seems to work quite well, my mother is constantly in "here own zone" and not registered as home in my house. The 10 second refresh of location is now gone. 

The home assistant app still sends periodical location updates (around every 10 minutes) so as an experiment to get rid of them when beeing stationary I disabled the background reporting of location in the app and kept the three other options (zone, significant location change and on request). So far location tracking seems to work well enough without this turned on and iCloud3 keeps working and reporting my whereabouts.

Thanks once more for working on this and giving support 👍
